### ISSUE TITLE
Add an AVG value to mods with min/max attributes

### ISSUE THREAD

**Issue by trackpete** [trackpete/exiletools-indexer#102]
For example:

```
            "Adds #-# Cold Damage": {
              "min": 12,
              "max": 25
            },
```

Should be:

```
            "Adds #-# Cold Damage": {
              "min": 12,
              "max": 25,
              "avg": 18.5
            },
```

**Comment by trackpete** [comment]
This is done! Any value that has a `min` and `max` attribute will now have a calculated `avg` attribute extended to two places past the decimal. This is done simply by `(min + max) / 2` on any field.

Note that this should also be applied to Properties and pseudo fields and things like that - so you'll get an `avg` field for Total Elemental Damage, etc.

**Comment by thirdy** [comment]
Great! this would certainly help working on the search terms I'm working on

On Sun, Feb 28, 2016 at 2:39 AM, Pete Waterman <notifications@github.com>
wrote:

> Closed #102 <https://github.com/trackpete/exiletools-indexer/issues/102>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/trackpete/exiletools-indexer/issues/102#event-569138142>
> .
>



-- 
--
http://sites.google.com/site/thirdyderivera/
